rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -32,16 +32,44 @@ def is_recording_enabled(recording: RecordingStream | None) -> bool:
 LogSinkLike = Union[GrpcSink, FileSink]
 
 
-def _tee(
+def attach_sinks(
     *sinks: LogSinkLike,
     default_blueprint: BlueprintLike | None = None,
     recording: RecordingStream | None = None,
 ) -> None:
     """
     Stream data to multiple different sinks.
 
-    Duplicate sinks are not allowed. For example, two [`GrpcSink`]s that
+    Duplicate sinks are not allowed. For example, two [`rerun.GrpcSink`][]s that
     use the same `url`.
```

Comment:
here and on `recording_stream`:

what happens when I do anyways? (docstring should tell ideally)

Reply:
it throws a `ValueError`, added to docstrings